@navikt/core/css/alert.css

```
@@ -104,12 +83,9 @@
 
 .navds-alert--warning > .navds-alert__icon {
   color: var(--navds-alert-color-warning-icon);
-  background: radial-gradient(
-    circle,
-    var(--navds-alert-color-warning-icon-background) 50%,
-    0,
-    transparent
-  );
+  box-shadow: inset 10px 10px 0 0 var(--navds-alert-color-warning-background),
+    inset 10px -4px 0 0 var(--navds-alert-color-warning-background),
+    inset 14px 0 0 0 var(--navds-alert-color-icon-background);
```

Før hadde utropstegnet i warning-ikonet samme farge som tekst for å få god nok kontrast med den orange bakgrunnen. Er det noe vi går bort fra?

---

Ja designet har nå hvit ! og mørkere ikon orange-500->orange-600 for å dekke kontrastkravet

---

Skjønner!

Jeg prøvde meg på en annen metode, men jeg vet ikke om denne ble så mye bedre
```suggestion
background: linear-gradient(white, white) center / 16% 60% no-repeat;
```